web/client/components/mapcontrols/search/SearchBar.jsx

```diff
@@ -13,6 +13,7 @@ const LocaleUtils = require('../../../utils/LocaleUtils');
 const DropdownToolbarOptions = require('../../misc/toolbar/DropdownToolbarOptions');
 const CoordinateEntry = require('../../misc/coordinateeditors/CoordinateEntry');
 const Toolbar = require('../../misc/toolbar/Toolbar');
+const AdvancedSearch = require('./AdvancedSearch').default;
```

Comment:
SearchBar is used both for map search and for GeoCoder search. I think this is wrong. I suggest to generalize this or separate the two components.

Reply:
Generalize in what way? AdvancedSearch tools and SearchBar parts are controlled with props. Both can enable the parts of the component that are appropriate for them. SearchBar is also used for managers(user, group, context)

Reply:
I should generalize the SearchBar with buttons (other tools) and a panel to render as children. Then I should provide one implementation with the buttons and the tools  of the map and another with the buttons of the home page and with the advancedsearch as children

Reply:
sorry but yesterday github had some issues and you didn't received this comment